marconi/app/Main.hs

```diff
@@ -62,13 +64,16 @@ import Marconi.Logging (logging)
 --     > select slotNo, datumHash, datum from kv_datumhsh_datum where slotNo = 39920450;
 --     39920450|679a55b523ff8d61942b2583b76e5d49498468164802ef1ebe513c685d6fb5c2|X(002f9787436835852ea78d3c45fc3d436b324184
 
+type MambaAddresses = NonEmpty.NonEmpty (C.AddressInEra  C.ShelleyEra)
```

Comment:
It's not Mamba specific. Rename to `TargetAddresses`? Also, I'm expecting the list to be empty since it shouldn't be mandatory to have a list of addresses.

Reply:
the MambaAddresses is used as Maybe MambaAddresses in the `Options`, Cli Options.  It seemed  more descriptive to Either have Nothing, or a nonEmpty list. But I'll be happy to change it?

Reply:
@koslambrou I think this is the only remaining open question. Currently, is Maybe NonEmptyList,  did you still want me to change it?

Reply:
Ah now I understand. No need to change, it's fine :)